packages/@aws-cdk/aws-lambda/lib/event-source-mapping.ts

```diff
@@ -351,4 +362,10 @@ export enum StartingPosition {
    * always read the most recent data in the shard
    */
   LATEST = 'LATEST',
+
+  /**
+   * Start reading from a position defined by a time stamp.
```

Comment:
This documentation should mention that if supplied, `startingPositionTimestamp` must be set

Reply:
Added